docs/app/Views/Layouts.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { Header, Segment, Container, Divider, Card } from 'semantic-ui-react'
+
+const LayoutsExample = () => (
+  <Container as='div'>
+    <Segment masthead vertical>
+      <Container as='div'>
+        <Header as='h1'> Layouts
+          <div className='sub header'>Examples of using Semantic UI React</div>
+        </Header>
+        <Divider hidden />
+      </Container>
+    </Segment>
+    <Container as='div' className='main intro'>
```

Comment:
Let's also remove `className='main intro'`, we shouldn't have any extra classNames on the layouts as we have no additional styles to apply.  If the layouts require additional custom styling on top of the default theme, then we'll have to reconsider how we go about that.